cli/cmd/compose/pull.go

```diff
@@ -40,11 +43,19 @@ func pullCommand(p *projectOptions) *cobra.Command {
 		Use:   "pull [SERVICE...]",
 		Short: "Pull service images",
 		RunE: func(cmd *cobra.Command, args []string) error {
+			if opts.noParallel {
+				logrus.Warn("`--no-parallel` flag is deprecated and will be ignored")
```

Comment:
Do we want a log here or just to write something to stderr?

Reply:
I would assume logrus "warn" is designed for this purpose, but no strong opinion here
what does docker/cli for such cases ?

Reply:
Looks like it uses logrus: https://github.com/docker/cli/blob/88c6089300a82d3373892adf6845a4fed1a4ba8d/cli/command/container/opts.go#L345

I was thinking that a log with timestamp, etc. wasn't really ideal but consistency is more important.

Reply:
we already have a comparable DEPRECATION message on compose command, will align to it